src/tests/assert.ts

```diff
@@ -25,6 +25,20 @@ export module Assert {
 		}
 	}
 
+	export function posInSetIsDecremental(elementIds: string[]) {
+		let elementsInExpectedPosInSetOrder = elementIds.map(id => {return{name: id, elem: document.getElementById(id) }; });
```

Comment:
You shouldn't need a return in the map here, you can do something like
id => {name: id, elem: document.getElementById(id)}